pandas/core/arrays/datetimelike.py

```diff
@@ -36,6 +36,8 @@
 def _make_comparison_op(cls, op):
     # TODO: share code with indexes.base version?  Main difference is that
     # the block for MultiIndex was removed here.
+
+    # @ops.unpack_and_defer
```

Comment:
Using this here leads to recursion errors, related to the fact that this only returns NotImplemented for ABCDataFrame.  I think this will be easier to resolve after the change to composition.